_i18n/en/resources/moneropedia/mnemonicseed.md

```diff
@@ -1,19 +1,24 @@
 ---
 terms: ["mnemonic-seed", "mnemonic"]
-summary: "a 13 or 25 word phrase used to backup a Monero account, available in a number of languages"
+summary: "a 13, 14 or 25 word phrase used to backup a Monero wallet"
 ---
 
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 ### The Basics
 
-A 13 or 25 word phrase used to backup a Monero account, available in a number of languages. This 25-word phrase (13 words in the case of MyMonero) has all the information needed to view and spend funds from a Monero @account.
+A 13, 14 or 25 word phrase used to backup a Monero @wallet, available in a number of languages. This phrase contains all the information needed to backup and restore a wallet. You should never share it, because anyone that has access to it will be able to view and spend funds from the wallet.
 
-### In-depth Information
+### How to store the mnemonic seed safely
 
-In the official wallet, the mnemonic seed comprises 25 words with the last word being used as a checksum. Those words correspond to a 256-bit integer, which is the account's *private* @spend-key. The *private* @view-key is derived by hashing the private spend key with Keccak-256, producing a second 256-bit integer. The corresponding *public* keys are then derived from the private keys.
+The most recommended way to store your mnemonic seed is to write two copies of it on a piece of paper, using a pen. Never store your mnemonic seed on a computer or any internet connected device. Don't store it on USB drives, email or cloud drives. Don't make screenshots or take photos.
```

Comment:
i'd go with the saying "if it's not in three places it doesn't exist", seed offset is also a possibility,, i wouldnt recommend multisig at the moment due to the current UX but we have RINO that could be mentioned,,  touch on some methods resistant against water/fire e.g. stamping them into washers secured between a nut and bolt

Reply:
I don't like the idea of telling people how to store their seed. Also not sure why this is being changed, we already have a page on secure storage


writing on paper with pen sounds great, until 
1. someone finds your non-encrypted, plain text piece of paper and takes a picture
2. It is exposed to fire / water / dog ate my homework / lost in storage / found by your evil maid
3. someone produces a rubbing from the next page on the notepad
4. you get keylogged while typing it in to restore
5. etcetc


i dont understand why people think its safe. 
much easier to steal or make a copy of paper than to decrypt an encrypted backup.

personally, i think using a password manager with or without an airgapped device is infinitely more secure than pen and paper. add a never-stored seed offset for redundacy. 

https://www.getmonero.org/resources/user-guides/securely_purchase.html

```
save it as a PDF or text on a USB stick, burn it to CD/DVD, etc. Most likely you will want at least two or three copies, stored securely in different locations. If storing digitally, encrypt everything with a strong password. If storing on paper, do not show the wallet to anyone else who can memorize your 25 word key, or take a picture of the wallet without your permission. Sending someone a picture of the wallet is the same as giving away all of your funds.

Whichever method you chose, be sure there's no copy of the Monero wallet left over on the device you used. You may need to securely delete the Monero wallet if you saved it to a disk, or make sure your printer does not save a copy in memory.
```

Reply:
Agree on not providing advice here in such a brief paragraph (that contradicts the advice elsewhere).  The excerpt from securely purchase is sane. So a nack on hints of storage in this article.. its just for the different seed implementations